src/main/java/org/ossgang/commons/observables/operators/OperatorUtils.java

```diff
@@ -0,0 +1,28 @@
+package org.ossgang.commons.observables.operators;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.IntStream;
+
+import static java.util.stream.Collectors.toList;
+import static java.util.stream.Collectors.toMap;
+
+public final class OperatorUtils {
```

Comment:
package-private maybe?

Reply:
Yes, so methods can be public and we don't have a visibility leak in case we forgot for new methods